### PR TITLE
Mark tab as selected

### PR DESCRIPTION
**Pull request by leggsimon**
Allows selected item to be configured

### CODE REVIEW

File: navigation-config.js [hunk 1]
```diff
@@ -1,47 +1,53 @@
...
+		trackable: 'content-distribution',
+	},
+	{
+		name: 'Usage Reports',
+		href: '/usage',
+		trackable: 'usage-reports',
+	},
+	{
+		name: 'Licence Administration',
+		href: '#', // TODO
```
Comment (thurstontye) [inline navigation-config.js:19]: This just needs to be `https://licence-admin.ft.com/licences/[licenceId]/users` using whatever the `licenceId` we are using in `window.location.pathname` right?

**Review by thurstontye — APPROVED** [review]
LGTM 👍

**Review by dapenguin — APPROVED** [review]
👍

File: navigation-config.js (hunk 1 above)
Comment (leggsimon) [reply navigation-config.js:19]: yep, will address in next PR

**Pull request merged**